rpm/selinux_springboot.spec

```diff
@@ -0,0 +1,52 @@
+Name:		springboot-selinux
+Version:	#{version}#
+Release:        1%{?dist}
+Summary:	SELinux policy module for Springboot applications
+License:	GPLv2
+URL: 		https://github.com/hubertqc/selinux_springboot
+Source0:	https://github.com/hubertqc/selinux_springboot/archive/refs/tags/v%{version}.tar.gz
+BuildArch:	noarch
+
+BuildRequires:	selinux-policy-devel
+BuildRequires:	make
+
+Requires:	selinux-policy-devel
+Requires:	selinux-policy-targeted
+Requires:	policycoreutils
+
+%description
+SELinux policy module to confine Springboot applications started using systemd.
+The systemd service unit name must start with springboot, the start script must be 
+assigned the springboot_exec_t SELinux type.
+The Springboot application will run in the springboot_t domain.
+
+%prep
+%setup -q -n selinux_springboot-%{version}
+
+%build
+
+mkdir -p -m 0755 %{buildroot}/usr/share/selinux/packages/targeted
+mkdir -p -m 0755 %{buildroot}/usr/share/selinux/devel/include/contrib
```

Comment:
Maybe more in application sub directory. Contrib seems to be use by system delivered or base policy delivered apps.

Reply:
I had done that at first, but the directory contains one solitary file...

Reply:
![image](https://user-images.githubusercontent.com/10505948/192223722-47d9e56a-d675-472f-bd10-8026ebd942b0.png)
As an example

Reply:
that's why I proposed that.

Reply:
Ok for apps subir